src/main/scala/edu/ie3/simona/ontology/messages/services/WeatherMessage.scala

```diff
@@ -33,7 +36,7 @@ object WeatherMessage {
     *   Longitude of the requested location
     */
   final case class RegisterForWeatherMessage(
-      requestingActor: ActorRef,
+      requestingActor: ActorRef[ParticipantAgent.Request],
```

Comment:
Could also be `ActorRef[RegistrationResponseMessage]`, to narrow down the type more

Reply:
This is not possible, since we save this reference in the state data, in order to send `DataProvision` messages later.

Reply:
Ah, yes. Ok.